tests/test_DaysTillNextFullMoon.py

```diff
@@ -0,0 +1,12 @@
+import unittest
+
+from AstroCal.control.control import getDaysTillFullMoon
+
+
+class TestDaysTillFullMoon(unittest.TestCase):
+    def test_DaysTillFullMoon_WhenSuccessful(self):
```

Comment:
Failed Test:
ERROR: test_DaysTillFullMoon_WhenSuccessful (test_DaysTillNextFullMoon.TestDaysTillFullMoon)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\Users\12503\Documents\GitHub\AstroCal/tests\test_DaysTillNextFullMoon.py", line 8, in test_DaysTillFullMoon_WhenSuccessful
    self.assertEqual(getDaysTillFullMoon(2025, 6, 1, -7), 10, "Test Fail")
  File "C:\Users\12503\Documents\GitHub\AstroCal\AstroCal\control\control.py", line 129, in getDaysTillFullMoon
    fmoon_year_utc, fmoon_month_utc, fmoon_day_utc, fmoon_hours_utc, fmoon_minutes_utc, _ = getDateOfNextFullMoon_UTC(
ValueError: not enough values to unpack (expected 6, got 3)